install/abstract-activable.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * @package Polylang
+ */
+
+/**
+ * A generic (de)activation class compatible with multisite.
+ *
+ * @since 3.8
+ */
+abstract class PLL_Abstract_Activable {
+	/**
+	 * (De)Activation for all blogs.
+	 *
+	 * @since 3.8
```

Comment:
I like to keep the original `@since` even when the method is moved to another class.
```suggestion
	 * @since 1.2
```
There are other cases in other classes.

Reply:
Fixed in [ee66d86](https://github.com/polylang/polylang/pull/1704/commits/ee66d862f32c9c9b0dce15b0dc478589373ab454).